junebug/channel.py

```diff
@@ -201,6 +201,19 @@ def get_all(cls, redis):
         channels = yield redis.smembers('channels')
         returnValue(channels)
 
+    @classmethod
+    @inlineCallbacks
+    def start_all_channels(cls, redis, config, parent):
+        '''Ensures that all of the stored channels are running'''
+        for id in (yield cls.get_all(redis)):
+            try:
+                yield cls.from_id(redis, config, id, parent)
```

Comment:
I see. Cool, lets leave it as is. Not too important, but maybe as a future thing, we should consider making `channel.start` a no-op for started channels.